docs/README.md

```diff
@@ -6,9 +6,10 @@ Please update any links to point to the new website instead.
 ## About
 
 This directory currently holds a series of markdown files documenting how to install, use
-and develop Synapse, the reference Matrix homeserver. The documentation is readable directly
-from this repository, but it is recommended to instead browse through the
-[website](https://matrix-org.github.io/synapse) for easier discoverability.
+and develop Synapse, a Matrix "homeserver" implementation developed by the matrix.org core 
+team. The documentation is readable directly from this repository, but it is recommended 
```

Comment:
I don't think this is the right place to describe what Synapse is.

```suggestion
and develop Synapse. The documentation is readable directly from this repository, but it is recommended 
```

Reply:
I agree -> Changed it!